components/core/test-locust.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+echo Installing locustio
+python3 -m pip install locustio
+echo Installing Docker
+if [[ "$OSTYPE" == "darwin"* ]];
+then
+    brew install docker
+else
+    sudo apt-get install docker
+fi
+echo Building and running container
+cd ../..
+echo Building from: $PWD...
+docker-compose -f docker-compose.yml up --build
```

Comment:
Throw an error if this command fails, and report the developer that mostly has to install docker and docker-compose in the working machine pointing to some relevant docker installation guide.